torch/csrc/jit/autodiff.cpp

```diff
@@ -318,7 +318,7 @@ static std::vector<Value*> gradientForNode(Node* node, ArrayRef<Value*> grad_val
       }
       return {sizes.at(dim) > 1 ? grads.at(0) : grads.at(0).unsqueeze(dim), nullptr};
 
-    } else if (node->matches("aten::cat(Tensor[] tensors, int dim) -> Tensor", /*const=*/attr::dim)) {
+    } else if (node->matches("aten::cat(Tensor[] tensors, int dim) -> Tensor", /*const_inputs=*/attr::dim)) {
```

Comment:
Oh noooo this will be so terribly verbose... Can we please disable this rule?

Reply:
I can disable it, but I would consider again whether it's a good idea. The thing is that it serves the exact same purpose as keyword arguments in Python, in that it's a safety against the eventual situation where you swap two argument and forget one call site where you don't swap the values too. For example, there was a bug in the export code just a few days where someone swapped the`count` and `value` arguments to `std::vector`. If that person had written
```
std::vector<int> v(/*value=*/1000000000000, /*count=*/1);
```
clang tidy here would warn you that `std::vector` has no constructor that takes a value and a count, and that you probably meant to write `/*count=*/1`, /*value=*/`. You just saved yourself from an OOM.

Do you really want to disable it? Again, I'm ok with it, just considering